### PR TITLE
Feature/cvsb 447

### PR DESCRIPTION
**Pull request by VioricaPatrascu**

### CODE REVIEW

File: src/models/TestTypesDto.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const AWS = require('aws-sdk')
+const config = require('../config/config')
+
+const dbClient = new AWS.DynamoDB.DocumentClient(
+  (config.ENV === 'local') ? { region: config.OFFLINE.DYNAMODB_REGION, endpoint: config.OFFLINE.DYNAMODB_ENDPOINT } : {})
+
+class TestTypesDto {
```
Comment (akothiyal-deloitte) [inline src/models/TestTypesDto.js]: rename to DAO as this doesnt seem like a DTO please

File: tests/models/TestTypesDtoMock.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+class TestTypesDtoMock {
```
Comment (akothiyal-deloitte) [inline tests/models/TestTypesDtoMock.js]: rename to DAOMock instead of DTOMOck pls

File: src/services/testTypesService.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  insertTestTypesList (testTypesItems) {
+    return this.testTypesDto.createMultiple(testTypesItems)
+      .then(data => {
+        if (data.UnprocessedItems) {
+          return data.UnprocessedItems
+        }
+      })
+      .catch((error) => {
```
Comment (akothiyal-deloitte) [inline src/services/testTypesService.js]: log the error pls

File: src/services/testTypesService.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+  }
+
+  deleteTestTypesList (testTypesItemKeys) {
+    return this.testTypesDto.deleteMultiple(testTypesItemKeys)
+      .then((data) => {
+        if (data.UnprocessedItems) {
+          return data.UnprocessedItems
+        }
+      })
+      .catch((error) => {
```
Comment (akothiyal-deloitte) [inline src/services/testTypesService.js]: log the error pls

File: src/functions/getTestTypes.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  const testTypesService = new TestTypesService(testTypes)
+
+  return testTypesService.getTestTypesList()
+    .then((data) => {
+      return {
+        statusCode: 200,
+        body: JSON.stringify(data)
+      }
+    })
+    .catch((error) => {
```
Comment (akothiyal-deloitte) [inline src/functions/getTestTypes.js]: log the error pls

File: src/services/testTypesService.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+  getTestTypesList () {
+    return this.testTypesDto.getAll()
+      .then(data => {
+        if (data.Count === 0) {
+          throw new HTTPResponseStatus(404, 'No resources match the search criteria.')
+        }
+        return data.Items
+      })
+      .catch(error => {
```
Comment (akothiyal-deloitte) [inline src/services/testTypesService.js]: log the error pls

File: package.json [hunk 7]
```diff
@@ -24,15 +28,16 @@
...
-    "supertest": "^3.3.0",
-    "standard": "^12.0.1"
+    "standard": "^12.0.1",
+    "supertest": "^3.3.0"
   },
   "dependencies": {
     "aws-sdk": "^2.332.0",
-    "expect": "^23.6.0",
-    "serverless": "^1.32.0"
+    "chai": "^4.2.0"
```
Comment (acozianu) [inline package.json]: Please move chat to devDependencies

File: serverless.yml [hunk 8]
```diff
@@ -10,8 +10,8 @@ plugins:
   - serverless-offline
 
 functions:
-  getDefects:
-    handler: handler.getTestTypes
+  getTestTypes:
+    handler: src/handler.getTestTypes
```
Comment (acozianu) [inline serverless.yml]: Based on Temi's request, the function from "getTestTypes" to "handler"

File: package.json (hunk 7 above)
Comment (VioricaPatrascu) [reply package.json]: Done

File: serverless.yml (hunk 8 above)
Comment (VioricaPatrascu) [reply serverless.yml]: Done

**Pull request merged**